index.html

```diff
@@ -1560,6 +1560,38 @@
   </div>
   <!-- ends Join files together -->
 
+
+  <!-- Split file into segments -->
+  <span data-toggle="modal" data-target="#segment_file"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Split one file into several smaller segments">Split file into segments</button></span>
+  <div id="segment_file" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Split file into segments</h3>
+          <p><code>ffmpeg -i <i>input_file</i> -c copy -map 0 -f segment -segment_time 60 -reset_timestamps 1 <i>output_file-%03d.mkv</i></code></p>
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-i <i>input_file</i></dt><dd>takes in a normal file</dd>
+            <dt>-c copy</dt><dd>use stream copy mode to re-mux instead of re-encode.</dd>
+            <dt>-map 0</dt><dd>Tells ffmpeg to map all streams of the input to the output.</dd>
+            <dt>-f segment</dt><dd>use <a href="http://www.ffmpeg.org/ffmpeg-formats.html#toc-segment_002c-stream_005fsegment_002c-ssegment">segment muxer</a> for generating the output.</dd>
+            <dt>-segment_time 60</dt><dd>Set duration of each segment (in seconds). This example creates segments with max. duration of 60s each.</dd>
+            <dt>-reset_timestamps 1</dt><dd>Reset timestamps of each segment to 0. Meant to ease the playback of the generated segments.</dd>
+            <dt><i>output_file-%03d.mkv</i></dt>
+            <dd>
+            path, name and extension of the output file.<br />
+            In order to have an incrementing number in each segment filename, FFmpeg supports <a href="http://www.cplusplus.com/reference/cstdio/printf/">printf-style</a> syntax for a counter.
+            In this example, '%03d' means: 3-digits, zero-padded<br />
```

Comment:
Thank you! I also personally prefer XHTML, but in practice HTML5 is currently _the_ standard. We moved `ffmprovisr` some months ago.